This patch has been submitted and accepted by
upstream and can be removed when we bring in 
the  next version 1.2.9, which will be done
by 2016-Jun-01.

--- openscap-1.2.6/src/OVAL/fts_sun.h~1~	2016-03-16 10:49:51.181254807 -0700
+++ openscap-1.2.6/src/OVAL/fts_sun.h	2016-03-16 10:50:51.407098175 -0700
@@ -248,5 +248,9 @@
 __END_DECLS
 
 #endif /* fts.h */
+#else
+#if defined(__SVR4) && defined(__sun)
+#include <fts.h>
+#endif /* defined(__SVR4) && defined(__sun) */ 
 
 #endif /* HAVE_FTS_OPEN */
